crates/core/src/spk_client.rs

```diff
@@ -317,6 +372,13 @@ impl<I> SyncRequest<I> {
         SyncIter::<I, ScriptBuf>::new(self)
     }
 
+    /// Iterate over [`ScriptBuf`]s with corresponding [`Txid`] histories contained in this request.
+    pub fn iter_spks_with_expected_txids(
+        &mut self,
+    ) -> impl ExactSizeIterator<Item = SpkWithExpectedTxids> + '_ {
+        SyncIter::<I, SpkWithExpectedTxids>::new(self)
+    }
```

Comment:
question: the `iter_spks` fn above is not being used anymore, at least for `SyncRequest`, do we need to keep it?

Reply:
Removed.